src/agent/src/image_rpc.rs

```diff
@@ -47,6 +77,213 @@ impl ImageService {
         }
     }
 
+    fn build_oci_path(cid: &str) -> PathBuf {
```

Comment:
I wonder if this should return `Result<PathBuf>` for the case where `cid == ""`?